### PR TITLE
Standardize resource joining docs around the Proxy

### LINKED ISSUE #21851: Extract the "Joining Nodes..." and "Adding Nodes" guides into a separate subsection of the docs
## Applies To
- `docs/pages/management/guides/joining-nodes-*.mdx`
- `docs/pages/management/admin/adding-nodes.mdx`

## Details
Once we merge #21597, we'll have three "Joining Nodes" guides (in addition to the original token-based "Adding Nodes" guide), so I think we should do this:

- Add another section to "Manage your Cluster" called "Joining Nodes"
- Move all of our "Joining Nodes..." guides into this section, including the "Adding Nodes" guide from "Manage your Cluster > Admin Guides".
- Add redirects from the old locations of our "Joining Nodes"/"Adding Nodes" guides to the new locations
- Add an intro page to the "Joining Nodes" section that provides a quick overview of the different joining methods and indicates when you would use each one.

Otherwise, the guides re: adding Nodes are a little hard to find, and it's a bit difficult to tell when in a operator's time with Teleport they should be using these guides.

Since the Core Concepts page no longer uses the term "Node", we should also think about substituting this with a more appropriate term.

## Related Issues
- https://github.com/gravitational/teleport/issues/17556
- https://github.com/gravitational/teleport/issues/19335

### PR DESCRIPTION
**Pull request by ptgott**
Closes #21305
Closes #21851

- Replace instructions to connect Teleport services to the Proxy Service over port 3080, plus the Auth Service over port 3025, with instructions to connect to the Proxy Service on port 443.  Also replace examples.

- Edit Access Request plugin guides to remove the options to connect to the Auth Service directly.

- Split the old "Adding Nodes" guide to include separate instructions for joining Teleport services with tokens via the Proxy Service and via the Auth Service, and make it explicit that we recommend the Proxy Service approach.

  To accommodate this change, add a separate subsection of the docs re: registering Teleport services, including a new intro page for this section. (While I was at it, I thought I would rename "Node" to "service" to match our Core Concepts guide.)

### CODE REVIEW

File: docs/config.json [hunk 1]
```diff
@@ -419,13 +419,33 @@
       "title": "Manage your Cluster",
       "entries": [
         {
-          "title": "Admin Guides",
-          "slug": "/management/admin/",
+          "title": "Registering Teleport Services",
```
Comment (zmb3) [inline docs/config.json:422]: Personally, if I were looking for this doc, I'd be searching for something with "join" in the title. Do you think "registering services" is descriptive enough?

File: docs/config.json [hunk 2]
```diff
@@ -419,13 +419,33 @@
...
+              "title": "Via AWS EC2",
+              "slug": "/management/register-services/register-services-aws-ec2/",
+              "forScopes": ["oss", "enterprise"]
+            },
+            {
+              "title": "Via AWS IAM",
+              "slug": "/management/register-services/register-services-aws-iam/"
             },
+            {
+              "title": "Via a Token Using the Auth Service",
```
Comment (zmb3) [inline docs/config.json:439]: Friendly ping for @atburke who is working on Azure joining and should be aware of this reorganization.

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 3]
```diff
@@ -60,8 +60,7 @@ teleport:
   #
   # We recommend to use tools like `pwgen` to generate sufficiently random
   # tokens of 32+ byte length.
-  # you can also use auth server's IP, i.e. "10.1.1.10:3025"
-  auth_server: 10.1.1.10:3025
+  auth_server: 10.1.1.10:443
```
Comment (zmb3) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:63]:
```suggestion
  proxy_server: 10.1.1.10:443
```

With config version `v3` you must use `auth_server` for auth servers and `proxy_server` for proxy servers.

File: docs/pages/access-controls/compliance-frameworks/fedramp.mdx [hunk 4]
```diff
@@ -100,9 +99,9 @@ teleport:
   auth_token: xxxx-token-xxxx
 
   # Specify either the proxy address
-  proxy_server: teleport.example.com:3080
+  proxy_server: teleport.example.com:443
   # or the auth server address
-  auth_server: 10.1.1.10:3025
+  auth_server: 10.1.1.10:443
```
Comment (zmb3) [inline docs/pages/access-controls/compliance-frameworks/fedramp.mdx:104]: The auth server does not listen on port 443. I would revert this change.

File: docs/pages/access-controls/sso/adfs.mdx [hunk 5]
```diff
@@ -49,16 +49,16 @@ another mapping of `E-Mail-Addresses` to `UPN`.
 
 You'll also need to create a Relying Party Trust. Use the below information to
 help guide you through the Wizard. Note that for development purposes we recommend
-using `https://localhost:3080/v1/webapi/saml/acs` as the Assertion Consumer
+using `https://localhost:443/v1/webapi/saml/acs` as the Assertion Consumer
```
Comment (zmb3) [inline docs/pages/access-controls/sso/adfs.mdx:52]: Might want to leave this as 3080 since it explicitly says _for development purposes_.

Binding to port 443 for development purposes would require running with elevated privileges.

Though I'm not sure why our public docs need to have specifics on _for development purposes..._

File: docs/pages/deploy-a-cluster/helm-deployments/custom.mdx [hunk 6]
```diff
@@ -94,7 +94,7 @@ proxy:
         severity: INFO
     proxy_service:
       enabled: true
-      listen_addr: 0.0.0.0:3080
+      listen_addr: 0.0.0.0:443
```
Comment (zmb3) [inline docs/pages/deploy-a-cluster/helm-deployments/custom.mdx:97]: I'd check with @hugoShaka, but this might be a breaking change if the helm deployment uses a kube service on port 443 and the proxy service still listens on port 3080 on the backend.

File: docs/pages/kubernetes-access/discovery/azure.mdx [hunk 7]
```diff
@@ -233,7 +233,7 @@ teleport:
     token_name: "/tmp/token"
     method: token
   auth_servers:
-  - "teleport.example.com:3080"
+  - "teleport.example.com:443"
```
Comment (zmb3) [inline docs/pages/kubernetes-access/discovery/azure.mdx:236]: Let's update this to use config v3 while we're here. It's confusing to users to specify a proxy server under `auth_servers`.

File: docs/pages/machine-id/getting-started.mdx [hunk 8]
```diff
@@ -234,8 +234,8 @@ Replace the following fields with values from your own cluster.
 - `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command.
 - `destination-dir` is where Machine ID writes user certificates that can be used by applications and tools.
 - `data-dir` is where Machine ID writes its private data, including its own short-lived renewable certificates. These should not be used by applications and tools.
-- `auth-server` is the address of your Teleport Auth Server, for example
-  `auth.example.com:3025`. If your Machine ID host is in a different network
+- `auth-server` is the address of your Teleport Proxy Service, for example
+  `teleport.example.com:443`. If your Machine ID host is in a different network
   than your Auth Server, use the public web address of your Proxy Service
```
Comment (zmb3) [inline docs/pages/machine-id/getting-started.mdx:239]: This no longer reads correctly, since it basically says "use the proxy or use the proxy instead"

File: docs/pages/machine-id/guides/host-certificate.mdx [hunk 9]
```diff
@@ -177,7 +177,7 @@ First, create a basic configuration file using the following parameters:
      --join-method=token \
      --certificate-ttl=1h0m0s \
      --ca-pin=(=presets.ca_pin=) \
-     --auth-server=auth.example.com:3025
+     --auth-server=auth.example.com:443
```
Comment (zmb3) [inline docs/pages/machine-id/guides/host-certificate.mdx:180]:
```suggestion
     --auth-server=proxy.example.com:443
```

Similar to previous comments, let's not confuse readers by giving the impression that auth listens on 443.

File: docs/pages/machine-id/guides/host-certificate.mdx [hunk 10]
```diff
@@ -207,7 +207,7 @@ First, create a basic configuration file using the following parameters:
      --join-method=iam \
      --certificate-ttl=1h0m0s \
      --ca-pin=(=presets.ca_pin=) \
-     --auth-server=auth.example.com:3025
+     --auth-server=auth.example.com:443
```
Comment (zmb3) [inline docs/pages/machine-id/guides/host-certificate.mdx:210]:
```suggestion
     --auth-server=proxy.example.com:443
```

File: docs/pages/machine-id/guides/host-certificate.mdx [hunk 11]
```diff
@@ -246,8 +246,8 @@ Replace the following fields with values from your own cluster:
     - `ca-pin` is the CA Pin for your Teleport cluster, and is output by the `tctl bots add` command.
     - `data-dir` is where Machine ID writes its private data, including its own short-lived renewable certificates. These should not be used by applications and tools.
     - `certificate-ttl` is the option that will determine the time until expiration for any `tbot` issued certificates, including host certificates. The default TTL is for one hour, meaning that all generated certificates will expire after one hour's time.
-    - `auth-server` is the address of your Teleport Auth Server, for example
-      `auth.example.com:3025`. If your Machine ID host is in a different network
+    - `auth-server` is the address of your Teleport Proxy Service, for example
```
Comment (zmb3) [inline docs/pages/machine-id/guides/host-certificate.mdx:249]: See previous comment. I think this was more correct before.

File: docs/pages/machine-id/troubleshooting.mdx [hunk 12]
```diff
@@ -16,8 +16,8 @@ ERROR: renewable cert generation mismatch: stored=3, presented=2
 Subsequent connection attempts by the bot may see errors like the following:
 ```text
 ERROR: failed direct dial to auth server: auth API: access denied [00]
-"\tauth API: access denied [00], failed dial to auth server through reverse tunnel: Get \"https://teleport.cluster.local/v2/configuration/name\": Get \"https://example.com:3025/webapi/find\": x509: cannot validate certificate for example.com because it doesn't contain any IP SANs"
-"\tGet \"https://teleport.cluster.local/v2/configuration/name\": Get \"https://example.com:3025/webapi/find\": x509: cannot validate certificate for example.com because it doesn't contain any IP SANs"
+"\tauth API: access denied [00], failed dial to auth server through reverse tunnel: Get \"https://teleport.cluster.local/v2/configuration/name\": Get \"https://teleport.example.com:443/webapi/find\": x509: cannot validate certificate for example.com because it doesn't contain any IP SANs"
```
Comment (zmb3) [inline docs/pages/machine-id/troubleshooting.mdx:19]: Revert these. The actual error that people would see would be port 3025.

File: docs/pages/management/admin/labels.mdx [hunk 13]
```diff
@@ -107,7 +107,7 @@ On the host where you will run your Node, paste the following content into
 version: v3
 teleport:
   # The address of your Auth Service.
-  auth_server: tele.example.com:3025
+  auth_server: tele.example.com:443
```
Comment (zmb3) [inline docs/pages/management/admin/labels.mdx:110]: Revert this.

If you want, feel free to mention `proxy_server` first and offer auth as an alternative.

File: docs/pages/management/guides/docker.mdx [hunk 14]
```diff
@@ -87,7 +87,7 @@ $ docker run --hostname localhost --rm \
 $ docker run --hostname localhost --name teleport \
   -v ~/teleport/config:/etc/teleport \
   -v ~/teleport/data:/var/lib/teleport \
-  -p 3023:3023 -p 3025:3025 -p 3080:3080 \
+  -p 3023:3023 -p 3025:3025 -p 443:443 \
```
Comment (zmb3) [inline docs/pages/management/guides/docker.mdx:90]: I'd revert this unless you actually tested the guide with this config.

I suspect you probably want `-p 443:3080`, but that may require root.

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 15]
```diff
@@ -1,45 +1,68 @@
...
+## Step 1/3. Install Teleport
 
-On the host where you will run your Teleport Node, follow the instructions for
-your environment to install Teleport.
+Install Teleport on the host where you will run your Teleport service.
 
 (!docs/pages/includes/install-linux.mdx!)
 
-## Step 2/3. Join your Node to the cluster
+## Step 2/3. Join your service to the cluster
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:45]: Technically, `instance` would be a more correct term, as multiple services can run in a single Teleport process.

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 16]
```diff
@@ -1,45 +1,68 @@
...
 
-In this section, we will join your Node to the Teleport cluster by:
+In this section, we will join your Teleport service to the Teleport cluster by:
 
-- Obtaining information stored on the Auth Service
-- Starting Teleport on your Node with the information we obtained
+- Obtaining a CA pin to establish trust with the Auth Service
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:49]: CA pinning is only used when connecting directly to auth. If we're recommending joining via proxy then I'd probably remove this step.

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 17]
```diff
@@ -58,13 +81,13 @@ $ export CA_PIN=$(tctl status | awk '/CA pin/{print $3}')
 
 ### Generate a token
 
-Teleport only allows access to Nodes that have joined the cluster.
+Teleport only allows access to services that have joined the cluster.
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:84]:
```suggestion
Teleport only allows access to resources that have joined the cluster.
```

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 18]
```diff
@@ -58,13 +81,13 @@ $ export CA_PIN=$(tctl status | awk '/CA pin/{print $3}')
...
 
-Once a Node joins, it receives a host certificate signed by the cluster's
+Once a service joins, it receives a host certificate signed by the cluster's
 Auth Service. To receive a host certificate upon joining a cluster, a new
-Teleport host must present an **invite token**.
+Teleport host must present an **join token**.
 
-An invite token also defines which role a new host can assume within a cluster:
+An join token also defines which role a new host can assume within a cluster:
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:90]:
```suggestion
A join token also defines which role a new host can assume within a cluster:
```

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 19]
```diff
@@ -81,16 +104,15 @@ On your local machine, use the `tctl` tool to generate a new token. In the
 following example, a new token is created with a TTL of five minutes:
 
 ```code
-# Generate a short-lived invite token for a new node:
-$ export INVITE_TOKEN=$(tctl nodes add --ttl=5m --roles=node | grep "invite token:" | grep -Eo "[0-9a-z]{32}")
+$ export INVITE_TOKEN=$(tctl nodes add --ttl=5m --roles=node | grep "join token:" | grep -Eo "[0-9a-z]{32}")
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:107]: Maybe we should use `tctl tokens add` instead. This was originally written when SSH nodes were the only type of supported resource.

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 20]
```diff
@@ -81,16 +104,15 @@ On your local machine, use the `tctl` tool to generate a new token. In the
...
 $ echo $INVITE_TOKEN
 # (=presets.tokens.first=)
 
 # You can also list all generated non-expired tokens:
 $ tctl tokens ls
 # Token                            Type            Expiry Time
 # ------------------------         -----------     ---------------
-# (=presets.tokens.first=)         Node            25 Sep 18 00:21 UTC
+# (=presets.tokens.first=)         service            25 Sep 18 00:21 UTC
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:115]: `service` would never appear under the Type column like this.

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 21]
```diff
@@ -100,7 +122,7 @@ value by using the `--token` flag:
 
 ```code
 $ tctl nodes add --ttl=5m --roles=node,proxy --token=secret-value
-# The invite token: secret-value
+# The join token: secret-value
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:125]: Unless you've actually changed the tctl source code I would remove this.

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 22]
```diff
@@ -232,31 +186,31 @@ $ sudo teleport start \
    --auth-server=${ADDR?}
 ```
 
-As new Nodes come online, they start sending ping requests every few seconds to
+As new services come online, they start sending ping requests every few seconds to
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:189]:
```suggestion
As new services come online, they start sending heartbeats every few seconds to
```

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 23]
```diff
@@ -232,31 +186,31 @@ $ sudo teleport start \
...
 the Auth Service. This allows users to explore cluster membership and size.
 
-Run the following command on your local machine to see all of the Teleport Nodes
+Run the following command on your local machine to see all of the Teleport services
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:192]: `tctl nodes ls` only shows SSH nodes, so the docs are now incorrect here.

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 24]
```diff
@@ -232,31 +186,31 @@ $ sudo teleport start \
...
 in your cluster:
 
 ```code
 $ tctl nodes ls
 
-Node Name     Node ID                                  Address            Labels
+service Name     service ID                                  Address            Labels
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:198]: Revert this.

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 25]
```diff
@@ -281,21 +235,21 @@ $ tctl tokens ls
 # Token                                Role       Expiry Time (UTC)
 # -----                                ----       -----------------
 # (=presets.tokens.first=)     Proxy      never
-# (=presets.tokens.second=)     Node       17 May 16 03:51 UTC
+# (=presets.tokens.second=)    Service      17 May 16 03:51 UTC
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:238]: Revert

File: docs/pages/management/register-services/register-services-auth-service.mdx [hunk 26]
```diff
@@ -281,21 +235,21 @@ $ tctl tokens ls
...
-tokens used for adding Nodes to your cluster.
+tokens used for adding services to your cluster.
 
 </Admonition>
 
 In this example, the first token has a `never` expiry date because it is a
 static token configured via a config file.
 
-The token with the `Node` role was generated to invite a new Node to this
+The token with the `service` role was generated to invite a new service to this
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-auth-service.mdx:252]: There is no `service` role. Revert this.

File: docs/pages/management/register-services/register-services-aws-ec2.mdx [hunk 27]
```diff
@@ -132,14 +133,14 @@ file or environment variables. See
 [Specifying Credentials](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials)
 for details.
 
-## Step 2/4. Create the AWS Node joining token
+## Step 2/4. Create the AWS service joining token
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-aws-ec2.mdx:136]:
```suggestion
## Step 2/4. Create the AWS joining token
```

File: docs/pages/management/register-services/register-services-aws-ec2.mdx [hunk 28]
```diff
@@ -154,7 +155,7 @@ metadata:
   name: ec2-token
 spec:
   # use the minimal set of roles required
-  roles: [Node]
+  roles: [service]
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-aws-ec2.mdx:158]: Revert this.

File: docs/pages/management/register-services/register-services-aws-ec2.mdx [hunk 29]
```diff
@@ -279,7 +280,7 @@ version: v2
 metadata:
   name: ec2-multiaccount-token
 spec:
-  roles: [Node]
+  roles: [service]
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-aws-ec2.mdx:283]: revert this.

File: docs/pages/management/register-services/register-services-aws-ec2.mdx [hunk 30]
```diff
@@ -268,7 +269,7 @@ In the AWS account where your Teleport Auth Server is running:
 2. Attach this `teleport-AssumeRole-policy` to the IAM role your Teleport auth
    server has credentials for, see [Step 1.2](#step-12-attach-the-iam-policy).
 
-When creating the AWS Node joining token, include an allow rule for each foreign
+When creating the AWS service joining token, include an allow rule for each foreign
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-aws-ec2.mdx:272]:
```suggestion
When creating the AWS joining token, include an allow rule for each foreign
```

File: docs/pages/management/register-services/register-services-aws-iam.mdx [hunk 31]
```diff
@@ -84,14 +84,14 @@ configuration file or environment variables. For details, see the following guid
 
 [Specifying Credentials](https://aws.github.io/aws-sdk-go-v2/docs/configuring-sdk/#specifying-credentials)
 
-## Step 2/4. Create the AWS Node joining token
-Under the hood, Nodes will prove that they are running in your AWS account by
+## Step 2/4. Create the AWS service joining token
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-aws-iam.mdx:87]:
```suggestion
## Step 2/4. Create the AWS joining token
```

File: docs/pages/management/register-services/register-services-aws-iam.mdx [hunk 32]
```diff
@@ -103,17 +103,17 @@ metadata:
   name: iam-token
 spec:
   # use the minimal set of roles required
-  roles: [Node]
+  roles: [service]
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-aws-iam.mdx:106]: revert

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 33]
```diff
@@ -0,0 +1,253 @@
...
+
+An invite token also defines which role a new host can assume within a cluster:
+
+- `proxy`
+- `node`
+- `auth`
+- `app`
+- `db`
+- `kube`
+- `windowsdesktop`
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:55]: This list is incomplete. You're missing `bot` and `discovery` (and arguably `instance` too, but I'm not sure it's worth opening that can of worms here)

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 34]
```diff
@@ -0,0 +1,253 @@
...
+
+Administrators can generate tokens as they are needed. A token can be used
+multiple times until its time to live (TTL) expires.
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:58]: This is generally true, but it not the case if the token has the `bot` role.

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 35]
```diff
@@ -0,0 +1,253 @@
...
+$ tctl nodes add --ttl=5m --roles=node,proxy --token=secret-value
+# The invite token: secret-value
+```
+
+The value of `--token` should be cryptographically secure. For example, you can
+create an SHA256 hash to reduce the risk that a malicious actor will produce
+a token with the same value as yours:
+
+```code
+$ head -n 1 /dev/random | sha256sum
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:91]: Let's just remove this section. It's easier to inadvertently create a token without sufficient randomness this way. While this is possible, it's best if the user just let's teleport generate the token.

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 36]
```diff
@@ -0,0 +1,253 @@
...
+```
+</Details>
+
+### Start your service with the invite token
+
+Execute the following commands on the host where you will run your service so you
+can use the `INVITE_TOKEN` variable to start Teleport.
+
+```code
+$ export INVITE_TOKEN=<Var name="invite-token" />
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:125]: Lets use `JOIN_TOKEN` instead. An invite token is often used to refer to _user_ invites, and these tokens are not for users but for hosts.

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 37]
```diff
@@ -0,0 +1,253 @@
...
+IP address) and port `443`:
+
+```code
+$ export ADDR=mytenant.teleport.sh:443
+```
+
+<Details title="Ensuring the correct ports are opened">
+
+If you are have disabled TLS Routing, the Teleport Proxy Service listens on
+separate ports in order to register new services, rather than performing all
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:139]: I don't think this is correct. _Registration_ should still happen via 443/3080, but resource access would happen on other ports.

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 38]
```diff
@@ -0,0 +1,253 @@
...
+  This requires that an LB use a round-robin or a similar balancing algorithm.
+  Do not use sticky load balancing algorithms (a.k.a. "session affinity") with
+  Teleport Proxy Service instances.  
+
+</Details>
+
+Execute the following command on your service to add it to a cluster:
+
+```code
+$ sudo teleport start \
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:168]: How about `teleport configure` followed by `teleport start`?

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 39]
```diff
@@ -0,0 +1,253 @@
...
+   --roles=node \
+   --token=${INVITE_TOKEN?} \
+   --auth-server=${ADDR?}
+```
+
+As new services come online, they start sending ping requests every few seconds to
+the Auth Service. This allows users to explore cluster membership and size.
+
+Run the following command on your local machine to see all of the Teleport services
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:177]: This will not show all services.

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 40]
```diff
@@ -0,0 +1,253 @@
...
+in your cluster:
+
+```code
+$ tctl nodes ls
+
+service Name     service ID                                  Address            Labels
+---------     -------                                  -------            ------
+turing        d52527f9-b260-41d0-bb5a-e23b0cfe0f8f     10.1.0.5:3022      distro:ubuntu
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:185]: You'll only see port 3022 like this when services register via the auth server, but we've now standardized on registering via proxy, so this column would say `tunnel` instead.

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 41]
```diff
@@ -0,0 +1,253 @@
...
+dijkstra      c9s93fd9-3333-91d3-9999-c9s93fd98f43     10.1.0.6:3022      distro:debian
+```
+
+## Step 3/3. Revoke an invitation
+
+Tokens used for joining services to a cluster can be revoked before they are used.
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:191]: This says they can be revoked _before_ they are used, but then the text instructions you to use it and revoke it after it is used.

File: docs/pages/management/register-services/register-services-join-token.mdx [hunk 42]
```diff
@@ -0,0 +1,253 @@
...
+static token configured via a config file.
+
+The token with the `service` role was generated to invite a new service to this
+cluster. And the third token was generated to invite a new user to sign up.
+
+Tokens created via `tctl` can be deleted (revoked) via the `tctl tokens del`
+command. Run the following command to delete a token:
+
+```code
+$ tctl tokens del (=presets.tokens.first=)
```
Comment (zmb3) [inline docs/pages/management/register-services/register-services-join-token.mdx:244]:
```suggestion
$ tctl tokens rm (=presets.tokens.first=)
```

File: docs/pages/reference/config.mdx [hunk 43]
```diff
@@ -435,11 +435,11 @@ auth_service:
     # IP and the port to bind to. Other Teleport Nodes will be connecting to
     # this port (AKA "Auth API" or "Cluster API") to validate client
     # certificates
-    listen_addr: 0.0.0.0:3025
+    listen_addr: 0.0.0.0:443
```
Comment (zmb3) [inline docs/pages/reference/config.mdx:438]: Revert this.

File: docs/pages/server-access/guides/bpf-session-recording.mdx [hunk 44]
```diff
@@ -138,9 +138,9 @@ teleport:
   auth_token: (=presets.tokens.first=)
 
   # Replace with the address of the Teleport Auth Service
-  auth_server: 127.0.0.1:3025
+  auth_server: 127.0.0.1:443
```
Comment (zmb3) [inline docs/pages/server-access/guides/bpf-session-recording.mdx:141]: Revert

File: docs/pages/server-access/guides/ec2-discovery.mdx [hunk 45]
```diff
@@ -126,7 +126,7 @@ teleport:
     token_name: "/tmp/token"
     method: token
   auth_servers:
-  - "teleport.example.com:3080"
+  - "teleport.example.com:443"
```
Comment (zmb3) [inline docs/pages/server-access/guides/ec2-discovery.mdx:129]: Let's use config v3 here.

**Review by zmb3 — CHANGES_REQUESTED** [review]
It looks like a large find/replace has created a number of inconsistencies and errors. I caught a lot of them, but I'm sure I've missed some.

File: docs/pages/deploy-a-cluster/helm-deployments/custom.mdx (hunk 6 above)
Comment (hugoShaka) [reply docs/pages/deploy-a-cluster/helm-deployments/custom.mdx:97]: Nice catch :) The chart expects your proxy to listen on 3080 as the Kubernetes Service resource is already doing the 443 -> 3080 translation. 

Listening on port 3080 in a container is easier and better for security because creating listeners for ports < 1024 requires higher privileges. We're currently trying to harden the containers and chart and running on privileged ports might not be possible in a near future.

For those reasons, I strongly advise keeping the 3080 listener in kube-hosted configuration and maybe adding a comment explaining Kubernetes is redirecting 443 on the service to 3080 on the pod.

File: docs/pages/management/guides/docker.mdx (hunk 14 above)
Comment (hugoShaka) [reply docs/pages/management/guides/docker.mdx:90]: As a rule of thumb we try to avoid listening on < 1024 ports in docker container because it requires root or some other kind of privilege not set by default. We are working to stop running containers as root so this would break.

As the docker daemon is root, it should be possible to redirect the host's 443 port to the container 3080 and still provide a unified client experience as Zac suggests.

**Comment by ptgott** [comment]
@zmb3 Thanks for the comprehensive review! It looks like the sweeping find/replace approach was a bit misguided here. I am going to start by opening a new PR that reorganizes the "Joining Nodes" guides (and incorporate any feedback on this PR into that one), then see if I can take a more mindful, targeted approach toward updating the remaining docs.

File: docs/pages/management/register-services/register-services-auth-service.mdx (hunk 16 above)
Comment (ptgott) [reply docs/pages/management/register-services/register-services-auth-service.mdx:49]: I've split the original "Joining Nodes" guide to have separate guides for joining via the Auth/Proxy. This way, we don't need to have as much branching within the guide. This guide has to do with the Auth Service.
